readthedocs/templates/homepage.html

```diff
@@ -4,6 +4,10 @@
 
 {% load pagination_tags %}
 
+{% block extra_metas %}
+  <meta name="description" content="{% trans "Read the Docs simplifies technical documentation by automating building, versioning, and hosting for you. Build up-to-date documentation for the web, print, and eReaders on every version control push automatically." %}">
```

Comment:
eReaders is a bit odd here. Perhaps we should just say "for online and offline use" or something.